src/components/AddComment.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+export default function AddComment({ articleId, setComments, currentUser }) {
+  const [commentBody, setCommentBody] = useState("");
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [err, setErr] = useState("");
+
+  const handleSubmit = (e) => {
+    setIsSubmitted(true);
```

Comment:
I would rename this state to isSubmitting to illustrate you're tracking the progress of a submit, not whether a user has submitted successfully.